src/loop.zig

```diff
@@ -291,4 +291,11 @@ pub const SingleThreaded = struct {
             report("recv done, remaining events: {d}", .{old_events_nb - 1});
         }
     }
+
+    pub fn canCancel() bool {
```

Comment:
I'm not sure about adding a function into the API only for test purpose.
What about running the `cancel` test only under linux for now?